docker-compose.yml

```diff
@@ -17,7 +17,7 @@ services:
       - mongo
   dashboard:
     image: parseplatform/parse-dashboard
-    command: --dev --appId APPLICATION_ID --masterKey MASTER_KEY --appName affinity
+    command: --dev --appId APPLICATION_ID --masterKey MASTER_KEY --serverURL "${VUE_APP_PARSE_URL}" --appName affinity
```

Comment:
Where does it get the variable from? We are currently not linking the `.env*` into it. I guess this is for the gitpods?!

Reply:
Yeah, the variable should be set somewhere in the shell or passed directly to docker-compose via

```sh
VUE_APP_PARSE_URL=http://localhost:1337/parse docker-compose -d up dashboard
```